diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/EmptyBlock.kt

```diff
@@ -97,6 +83,9 @@ class EmptyBlock : Rule("block-structure") {
         }
     }
 
+    private fun isNodeEmpty(node: ASTNode) = listOf(LBRACE, RBRACE, WHITE_SPACE)
```

Comment:
You shouldn't copy-paste utility methods. Instead please extract this to some utility class (e.g. `ASTNodeUtils`).